actions/dependabot-automerge/action.yml

```
@@ -0,0 +1,53 @@
+name: 'Dependabot Automerge'
+description: 'Automatically approve and merge Dependabot PRs'
+inputs:
+  merge-strategy:
+    description: Merge strategy to use when enabling automerge
+    required: true
+    default: 'squash'
+  prod-semver-autoapprovals:
+    description: 'Semver levels for which to automatically approve the PR for production dependencies'
+    required: true
+    default: 'patch'
+  dev-semver-autoapprovals:
+    description: 'Semver levels for which to automatically approve the PR for development dependencies'
+    required: true
+    default: 'major,minor,patch'
+runs:
+  using: composite
+  steps:
+    - uses: actions/setup-python@v4
+      with:
+        python-version: '3.11'
+    - shell: bash
+      run: pip install click
+    - name: Dependabot metadata
+      id: dependabot-metadata
+      uses: dependabot/fetch-metadata@v1
+      with:
+        github-token: ${{ github.token }}
+    - name: Review PR
+      id: review-pr
+      shell: python
+      run: review_pr.py \
+          --semver-level "${{ steps.dependabot-metadata.outputs.update-type }}" \
+          --dependency-type "${{ steps.dependabot-metadata.outputs.dependency-type }}" \
+          --prod-semver-autoapprovals ${{ inputs.prod-semver-autoapprovals }} \
+          --dev-semver-autoapprovals ${{ inputs.dev-semver-autoapprovals }}
```

I was a bit hesitant about the approach you took for the parameters prefixes, but could not come up with a shorter/simpler idea. 

Apart maybe (maybe!!) if we would strip them here:
```suggestion
      shell: bash
      run: \
         SEMVER_LEVEL=${{ steps.dependabot-metadata.outputs.update-type }} \
         DEPENDENCY_TYPE=${{ steps.dependabot-metadata.outputs.dependency-type }} \
         
         python review_pr.py \
          --semver-level "${UPDATE_TYPE#version-update:semver-}" \
          --dependency-type "${DEPENDENCY_TYPE#direct:}" \
          --prod-semver-autoapprovals ${{ inputs.prod-semver-autoapprovals }} \
          --dev-semver-autoapprovals ${{ inputs.dev-semver-autoapprovals }}
```

---

Haha yeah, I just like to use the least amount of Bash possible since I'm never sure that I'm doing the right thing. I felt more "sure" about `<str>.removeprefix()`